jena-cmds/src/main/java/tdb2/tdbloader.java

```diff
@@ -92,6 +97,17 @@ else if ( loadername.matches("light") )
                 throw new CmdException("Not a boolean value: "+getValue(argStats));
             generateStats = super.hasValueOfTrue(argStats);
         }
+        
+        if ( super.contains(argSyntax) ) {
+            String syntax = super.getValue(argSyntax) ;
+            Lang lang$ = RDFLanguages.nameToLang(syntax) ;
+            if ( lang$ == null )
+                throw new CmdException("Can not detemine the syntax from '" + syntax + "'") ;
+            this.lang = lang$ ;
+        }
+        
+        if ( super.graphName != null )
```

Comment:
So if I specify `--syntax=LANG` but also specify `--graph=URL` then my syntax spec is ignored?

Reply:
Thanks for spotting that - default syntax should be set before `--syntax` is processed.